ultra/ultra/baselines/adapter.py

```diff
@@ -97,7 +95,6 @@ def observation_space(self):
             social_vehicle_shape = self.social_capacity * self.num_social_features
         return gym.spaces.Dict(
             {
-                # "images": gym.spaces.Box(low=0, high=1e10, shape=(1,)),
                 "low_dim_states": gym.spaces.Box(
                     low=-1e10,
                     high=1e10,
```

Comment:
Should we include images here?

Reply:
Also, do the values actually vary from -1e10 to 1e10? Or is this just for convenience.

Reply:
I'm not sure why they vary like this, I imagine it's for convenience, maybe @adreena can comment on this again? And we had a discussion about whether to include images for the baselines or or not (see the resolved conversation in Kimia's first review of this Pull Request). We decided against including them because the baselines do not use them whether they are in their observation or not. But did you have some reasons to keep the images in the baselines' observations @AlexLewandowski?

Reply:
Nope, I was just looking for clarification. Looks good then!